ocs_ci/utility/templating.py

```
@@ -181,14 +181,15 @@ def get_n_document_from_yaml(yaml_generator, index=0):
     raise IndexError(f"Passed yaml generator doesn't have index {index}")
 
 
-def dump_data_to_temp_yaml(data, temp_yaml):
+def dump_data_to_temp_yaml(data, temp_yaml, log=True):
```

Ruff (0.15.21)

[warning] Boolean default positional argument in function definition

(FBT002)